src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml

```diff
@@ -1,24 +1,27 @@
 @model OrchardCore.Media.ViewModels.DisplayMediaFieldViewModel
 @using OrchardCore.ContentManagement.Metadata.Models
-@using System.Text.RegularExpressions
+@using OrchardCore.Mvc.Utilities
+@using OrchardCore.Media
 @{
     var paths = Model.Paths ?? Array.Empty<string>();
     var mediaTexts = Model.Field?.MediaTexts ?? Array.Empty<string>();
+    var name = (Model.PartFieldDefinition.PartDefinition.Name + "-" + Model.PartFieldDefinition.Name).HtmlClassify();
 }
 @if (paths.Any())
 {
-    <div class="form-group row">
-        <label class="col-sm-2 col-form-label col-form-label-sm">@Model.PartFieldDefinition.DisplayName():</label>
-        <div class="col-sm-10">
+    <div class="form-group row field-type-mediafield field-name-@name">
+        <label class="col-xl-2">@Model.PartFieldDefinition.DisplayName()</label>
+        <div class="col-xl-10">
             @for (int i = 0; i < paths.Count(); i++)
             {
                 var path = paths[i];
                 var url = Orchard.AssetUrl(path);
                 var text = !string.IsNullOrEmpty(mediaTexts[i]) ? mediaTexts[i] : path;
-                var displayImg = Regex.IsMatch(path, @"(\.(gif|jpe?g|tiff?|png|webp|bmp)$)");
-                if (displayImg)
+                var extension = System.IO.Path.GetExtension(path);
```

Comment:
perhaps just pass the path the extension method, and it can handle get the extension.

Reply:
While the method name `IsImageExtension()` the expectation from user perspective is to pass the extension not the path